packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -229,6 +229,7 @@ const UsersProfileFn = ({terms, slug, classes}: {
     const ownPage = currentUser?._id === user._id
     const currentShowLowKarma = (parseInt(query.karmaThreshold) !== DEFAULT_LOW_KARMA_THRESHOLD)
     const currentIncludeEvents = (query.includeEvents === 'true')
+    terms.excludeEvents = !currentIncludeEvents && currentFilter !== 'events'
```

Comment:
I noticed that events were appearing in the user's posts list even though they should be excluded by default, so I fixed that as well